vita3k/io/src/io.cpp

```diff
@@ -244,12 +245,20 @@ SceUID open_file(IOState &io, const char *path, const int flags, const std::wstr
         return IO_ERROR(SCE_ERROR_ERRNO_ENOENT);
     }
 
-    const auto system_path = device::construct_emulated_path(device, translated_path, pref_path, io.redirect_stdio);
+    auto system_path = device::construct_emulated_path(device, translated_path, pref_path, io.redirect_stdio);
 
     // Do not allow any new files if they do not have a write flag.
     if (!fs::exists(system_path) && !can_write(flags)) {
-        LOG_ERROR("Missing file at {} (target path: {})", system_path.string(), path);
-        return IO_ERROR(SCE_ERROR_ERRNO_ENOENT);
+
+        const auto icase_path = device::attempt_icase_path(translated_path, system_path);
+
+        if(!icase_path.empty()) {
+            system_path = icase_path;
+        }
```

Comment:
format error here ^^

Reply:
Lol sry, it's still WIP I'll fix everything as I move along.